hazelcast/src/test/java/com/hazelcast/internal/usercodedeployment/impl/filter/UserCodeDeploymentBasicTest.java

```diff
@@ -19,28 +19,41 @@
 import com.hazelcast.config.Config;
 import com.hazelcast.config.UserCodeDeploymentConfig;
 import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.internal.util.FilteringClassLoader;
+import com.hazelcast.map.EntryProcessor;
 import com.hazelcast.map.IMap;
+import com.hazelcast.nio.serialization.HazelcastSerializationException;
 import com.hazelcast.test.HazelcastParallelParametersRunnerFactory;
+import com.hazelcast.test.HazelcastTestSupport;
 import com.hazelcast.test.TestHazelcastInstanceFactory;
 import com.hazelcast.test.annotation.ParallelJVMTest;
 import com.hazelcast.test.annotation.QuickTest;
-import com.hazelcast.internal.util.FilteringClassLoader;
+import org.junit.After;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 import usercodedeployment.ClassWithTwoInnerClasses;
+import usercodedeployment.DomainClassWithInnerClass;
+import usercodedeployment.EntryProcessorWithAnonymousAndInner;
+import usercodedeployment.IncrementingEntryProcessor;
+import usercodedeployment.blacklisted.BlacklistedEP;
+import usercodedeployment.whitelisted.WhitelistedEP;
 
 import java.util.Collection;
 
+import static com.hazelcast.test.starter.HazelcastStarterUtils.assertInstanceOfByClassName;
 import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
 
 @RunWith(Parameterized.class)
 @Parameterized.UseParametersRunnerFactory(HazelcastParallelParametersRunnerFactory.class)
 @Category({QuickTest.class, ParallelJVMTest.class})
-public class UserCodeDeploymentBasicTest extends UserCodeDeploymentAbstractTest {
```

Comment:
`UserCodeDeploymentAbstractTest` class is used by EE repo, so we should probably keep it.

Reply:
Did see the note in the PR description. My bad. Resolving this one.